### ISSUE TITLE
Completions in async functions appear incomplete

### ISSUE THREAD

**Issue by brettcannon** [davidhalter/jedi#1092]
From Microsoft/vscode-python#140:
```python
var1 = 42

async def foo():
    var2 = False
    var  # Only lists var1.
```

**Comment by davidhalter** [comment]
Can confirm that this is a bug.

**Comment by davidhalter** [comment]
Fixed in 8494164b22b7d4af2e8762c63e48c3eeffce66f5

Thanks for the report!
